onadata/libs/utils/csv_import.py

```diff
@@ -159,237 +160,273 @@ def failed_import(rollback_uuids, xform, exception, status_message):
     return async_status(FAILED, status_message)
 
 
-@use_master
-def submit_csv(username, xform, csv_file, overwrite=False):
-    """Imports CSV data to an existing form
+def validate_csv_file(csv_file, xform):
+    """Validates a CSV File
 
-    Takes a csv formatted file or string containing rows of submission/instance
-    and converts those to xml submissions and finally submits them by calling
-    :py:func:`onadata.libs.utils.logger_tools.safe_create_instance`
+    Takes a CSV Formatted file or sring containing rows of submission
+    data and validates that the file is valid enough to be processed.
 
-    :param str username: the subission user
-    :param onadata.apps.logger.models.XForm xfrom: The submission's XForm.
-    :param (str or file): A CSV formatted file with submission rows.
-    :return: If sucessful, a dict with import summary else dict with error str.
+    :param (str or file) csv_file: A CSV Formated file or string with
```

Comment:
'formatted' has 2 't's, it should also be lowercase here and above on line 166

Reply:
Changed `Formated` to `formatted`